blocks/init/components/heading/heading.php

```diff
@@ -8,14 +8,20 @@
 
 namespace Eightshift_Boilerplate\Blocks;
 
-$content = $attributes['content'] ?? '';
-$level   = isset( $attributes['level'] ) ? "h{$attributes['level']}" : 'h2';
+$heading = $attributes['heading'] ?? [];
+
+if ( empty( $heading ) ) {
```

Comment:
Since the default, if not set, is empty array, which is falsey value, you can omit the `empty` check and just have 

```php
f ( ! $heading ) {
```